test/util/h.js

```diff
@@ -3,6 +3,11 @@ import { createHyperscript } from 'slate-hyperscript'
 export default global.h = createHyperscript({
     blocks: {
         code: 'code',
+        definition: 'definition',
+        determ: 'definition_term',
+        demeaning: 'definition_meaning',
+        deexample: 'definition_example',
+        deseealso: 'definition_seealso',
```

Comment:
I know that we already have a two letter prefix (ex for exercise), but for some reason this is really confusing to me (possible because I've mostly seen de used as deserialization). I think I would prefer def as prefix here.